velox/expression/tests/ExpressionRunner.cpp

```diff
@@ -129,47 +207,76 @@ void ExpressionRunner::run(
       : deserializerPool;
   core::ExecCtx execCtx{pool.get(), queryCtx.get()};
 
-  RowVectorPtr inputVector;
-
-  if (inputPath.empty()) {
-    inputVector = std::make_shared<RowVector>(
-        deserializerPool.get(), ROW({}), nullptr, 1, std::vector<VectorPtr>{});
-  } else {
-    inputVector = std::dynamic_pointer_cast<RowVector>(
-        restoreVectorFromFile(inputPath.c_str(), deserializerPool.get()));
-    VELOX_CHECK_NOT_NULL(
-        inputVector,
-        "Input vector is not a RowVector: {}",
-        inputVector->toString());
-    VELOX_CHECK_GT(inputVector->size(), 0, "Input vector must not be empty.");
-  }
-
   fuzzer::InputRowMetadata inputRowMetadata;
   if (!inputRowMetadataPath.empty()) {
     inputRowMetadata = fuzzer::InputRowMetadata::restoreFromFile(
         inputRowMetadataPath.c_str(), pool.get());
   }
 
+  std::vector<fuzzer::InputTestCase> inputTestCases;
+  if (inputPaths.empty()) {
+    inputTestCases.push_back(
+        {std::make_shared<RowVector>(
+             deserializerPool.get(),
+             ROW({}),
+             nullptr,
+             1,
+             std::vector<VectorPtr>{}),
+         SelectivityVector(1)});
+  } else {
+    std::vector<std::string> inputPathsList = split(inputPaths);
+    std::vector<std::string> inputSelectivityPaths =
+        split(inputSelectivityVectorPaths);
+    for (int i = 0; i < inputPathsList.size(); i++) {
+      auto inputVector =
+          std::dynamic_pointer_cast<RowVector>(restoreVectorFromFile(
+              inputPathsList[i].c_str(), deserializerPool.get()));
+      VELOX_CHECK_NOT_NULL(
+          inputVector,
+          "Input vector is not a RowVector: {}",
+          inputVector->toString());
+      VELOX_CHECK_GT(inputVector->size(), 0, "Input vector must not be empty.");
+      if (inputSelectivityPaths.size() > i) {
```

Comment:
I may have missed something, but why could we have unmatched number of selectivity paths and input paths?

Reply:
there are 2 cases I am trying to handle, one where there are no input selectivity paths, the other where all inputs have one. However since I am allowing the user to input a comma separated list of paths, this check allows me to handle the case where the user only adds selectivity vector paths for some inputs.

The cmd line params don't offer the best user experience here, so I am definitely open to suggestions.

Reply:
@kagamiori you can either specify the folder where all files where generated by the fuzzer like before:

```
--fuzzer_repro_path /tmp/fuzzer_repro/velox_expressionVerifier_Gu1Seu/ --mode verify 
```

Or you can specify the input, etc explicitly like:

```
--input_paths /tmp/fuzzer_repro/velox_expressionVerifier_Gu1Seu/input_vector_0,/tmp/fuzzer_repro/velox_expressionVerifier_Gu1Seu/input_vector_1 --input_selectivity_vector_paths /tmp/fuzzer_repro/velox_expressionVerifier_Gu1Seu/input_selectivity_vector_0,/tmp/fuzzer_repro/velox_expressionVerifier_Gu1Seu/input_selectivity_vector_1 --sql_path /tmp/fuzzer_repro/velox_expressionVerifier_Gu1Seu/sql --mode verify 
```